app/src/main/java/com/yeonae/chamelezone/data/repository/place/PlaceRepositoryImpl.kt

```diff
@@ -68,34 +68,49 @@ class PlaceRepositoryImpl private constructor(private val remoteDataSource: Plac
         remoteDataSource.getKeyword(callBack)
     }
 
-    override fun modifyPlace(
+    override fun updatePlace(
+        placeNumber: Int,
+        images: List<String>,
         memberNumber: Int,
-        keywordNames: List<Int>,
-        name: String,
         address: String,
-        openingTimes: List<String>,
         phoneNumber: String,
         content: String,
         latitude: BigDecimal,
         longitude: BigDecimal,
-        images: List<String>,
+        imageNumber: List<Int>,
         callBack: PlaceCallBack<Boolean>
     ) {
-        remoteDataSource.modifyPlace(
+        remoteDataSource.updatePlace(
+            placeNumber,
+            images,
             memberNumber,
-            keywordNames,
-            name,
             address,
-            openingTimes,
             phoneNumber,
             content,
             latitude,
             longitude,
-            images,
+            imageNumber,
             callBack
         )
     }
 
+    override fun updateKeyword(
+        placeNumber: Int,
+        keywordNames: List<Int>,
+        placeKeywordNumber: List<Int>,
+        callBack: PlaceCallBack<Boolean>
```

Comment:
```suggestion
        callback: PlaceCallback<Boolean>
```
callback은 보통 이렇게 사용하는것 같습니다.

Reply:
수정했습니다
64476f8